app/routes/polls/components/PollEditor.js

```diff
@@ -0,0 +1,176 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Content } from 'app/components/Content';
+import NavigationTab, { NavigationLink } from 'app/components/NavigationTab';
+import Button from 'app/components/Button';
+import Icon from 'app/components/Icon';
+import { Link } from 'react-router';
+import {
+  TextInput,
+  SelectInput,
+  TextArea,
+  legoForm,
+  CheckBox
+} from 'app/components/Form';
+import { Form, Field, FieldArray } from 'redux-form';
+import { ConfirmModalWithParent } from 'app/components/Modal/ConfirmModal';
+import styles from './PollEditor.css'
+
+const renderOptions = ({ fields, meta: { touched, error } }) => {
+  return [
```

Comment:
I see this returns an array? Does that make any sense? If so; can't it be a direct return?

Reply:
Looks like its because it returns two independent elements, wrapping them in a div probably makes more sense.